package.json

```diff
@@ -36,7 +36,7 @@
     "depcheck": "npx npm-check --update"
   },
   "peerDependencies": {
-    "react": ">=17.0.0"
+    "react": ">=17.0.0 || 18"
```

Comment:
I think the previous version already allowed `18`, iirc for it to just be 17.X you need this; `>=17 < 18`. So not sure if we need this update 😄

Reply:
Tbh you're right, this package already works in 18. I just saw a lot of packages use this approach so mimicked them. In some ways I think this makes it clearer than we support 18, so would keep it, what do you think?

Reply:
No worries, let's go for it